msi/release.sh

```diff
@@ -111,35 +122,13 @@ echo "    Version: $VERSION"
 echo "    Build Type: $( (test -n "$CUSTOM" && test "$CUSTOM" -eq 1 && echo "Custom") || (test -n "$TEST" && test "$TEST" -eq "1" && echo "Test") || echo "Release")"
 echo
 
+# Generate the ReleaseNotes.html file
+../render-release-notes.sh --output msi/ --css usr/share/git/
```

Comment:
Again, if you follow my reasoning about avoiding the `cd`, this should use the options `--output . --css usr/share/git/` instead.